package.yaml

```diff
@@ -0,0 +1,45 @@
+Name:           gtkglext
+Version:        0.13.1.1
+License:        LGPL-2.1
+License-file:   COPYING
+Copyright:      (c) 2001-2010 The Gtk2Hs Team
+Author:         Duncan Coutts
+Maintainer:     gtk2hs-users@lists.sourceforge.net
+Stability:      provisional
+homepage:       http://projects.haskell.org/gtk2hs/
+bug-reports:    http://hackage.haskell.org/trac/gtk2hs/
+Synopsis:       Binding to the GTK+ OpenGL Extension
+Description:    GtkGLExt provides the GDK objects to support OpenGL rendering in GTK+,
+                and GtkWidget API add-ons to make GTK+ widgets OpenGL-capable.
+
+Category:       Graphics
+Tested-With:    GHC == 6.10.4, GHC == 6.12.3, GHC == 7.0.4, GHC == 7.2.1
+Build-Type:     Custom
+
+custom-setup:
+  setup-depends: 
+    - base
+    - Cabal
+    - gtk2hs-buildtools
+
+library:
+  dependencies:
+    - base
+    - glib
+    - pango
+    - gtk
+
+  exposed-modules:
+    - Graphics.UI.Gtk.OpenGL
+    - Graphics.UI.Gtk.OpenGL.Config
+    - Graphics.UI.Gtk.OpenGL.Context
+    - Graphics.UI.Gtk.OpenGL.Drawable
+    - Graphics.UI.Gtk.OpenGL.DrawingArea
+    - Graphics.UI.Gtk.OpenGL.General
+    - Graphics.UI.Gtk.OpenGL.Pixmap
+    - Graphics.UI.Gtk.OpenGL.Window
+  other-modules:
+    - Graphics.UI.Gtk.OpenGL.Types
+
+  pkgconfig-depends: gtkglext-1.0
+
```

Comment:
This seems like an unnecessary duplication of the `.cabal` file.  Can we just update the .cabal file instead of adding it?

Reply:
Thanks for the quick response Hamish

Not sure if you are familiar with `hpack` [1] - it is basically an (arguably) better format for writing cabal files. It is supported out of the box by `stack`. Basically a cabal file is generated on every build from the spec in the `package.yaml`, which is why I've added the `*.cabal` file to `.gitignore`.
Also I've kept the original cabal file as a reference as `.cabal.original` with eventual intention to remove it. 

[1] https://github.com/sol/hpack

Reply:
I am very far from convinced hpack is a good idea in general and the cabal flags seem to be missing from the `package.yaml` file.  Can you please split the hpack related changes into another PR?